src/caffe/test/test_gradient_based_solver.cpp

```diff
@@ -173,6 +174,9 @@ class GradientBasedSolverTest : public MultiDeviceTest<TypeParam> {
     if (momentum != 0) {
       proto << "momentum: " << momentum << " ";
     }
+    if (rms_decay != 0) {
```

Comment:
I think this could be handled in `RMSPropSolverTest::InitSolver()` since this detail is particular to the RMSProp solver alone.